dev-docs/source/RunningTheUnitTests.rst

```diff
@@ -103,31 +103,33 @@ Starting in your build folder (e.g. Mantid/Code/debug):
 
    -  Not possible with ctest.
 
-Visual Studio/ XCode note
-#########################
+Running Unit Tests from Within Visual Studio
```

Comment:
Can we call this section "Running Unit Tests With Visual Studio" ? It mentions both `ctest` from the command line and also Visual Studio itself.